stories/Form/FormTemplate/FormTemplate.stories.tsx

```diff
@@ -0,0 +1,122 @@
+import { Meta, Story } from "@storybook/react";
```

Comment:
```suggestion
import { Meta, StoryFn } from "@storybook/react";
```
as Story is deprecated